### PR TITLE
[EOSF-736] Ember-side Navbar cleanup (firefox username alignment, long username fix)

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose

Current there are two problems with the OSF navbar:

1. The username name text and gravtar are silghtly out of vertical alignment
2. Very long usernames (even though they are wrapped properly) can break the navbar into two unsightly rows.
Alignment problem:
screen shot 2017-07-05 at 3 19 15 pm
Row problem:
screen shot 2017-07-05 at 3 24 23 pm

## Changes

Simple CSS and HTML changes, now the navbar will collapse into a dropdown sooner and the alignment is fixed by removing classes from some of the navbar menu items and modifying the gravtar image border.

## Side effects

Should be limited to aesthetics. I used style tags where possible to avoid any unforeseeable interaction.

## Tests

For aesthetics changes I can't think of any meaningful tests.

## Ticket

https://openscience.atlassian.net/browse/EOSF-736

### CODE REVIEW

**Comment by alexschiller** [comment]
This PR should probably be associated separately with the EOSF side ticket: https://openscience.atlassian.net/browse/EOSF-736

File: addon/components/new-osf-navbar/template.hbs [hunk 1]
```diff
@@ -3,11 +3,11 @@
         <div class="container">
             <div class="navbar-header">
                 {{!TOGGLE NAVIGATION BUTTON - XS SCREEN}}
-                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#secondary-navigation" {{action 'closeSearch'}} aria-label={{t 'eosf.navbar.toggleSecondary'}}>
+                <a type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#secondary-navigation" {{action 'closeSearch'}} aria-label={{t 'eosf.navbar.toggleSecondary'}}>
```
Comment (jamescdavis) [inline addon/components/new-osf-navbar/template.hbs]: I think you need to add onclick= before the {{action 'closeSearch'}}

**Comment by jamescdavis** [comment]
@Johnetordoff can you rebase please?

**Comment by jamescdavis** [comment]
Closing in favor of https://github.com/CenterForOpenScience/ember-osf/pull/267

**Pull request closed without merging**